xAPI.md

```diff
@@ -1577,6 +1578,10 @@ if **all** of the items in that list match the learner's response. For example,
 
 In this example, ```foo[,]bar``` and  ```foo``` are correct learner responses; ```bar``` is not.
 
+The Correct Responses Pattern, if used, is intended to be an exhaustive list of possible correct responses. Where the criteria for a question are complex and correct responses cannot be exhaustively listed, Activity Providers are discouraged from using the "correct responses pattern" property.
+
+The Activity Provider is allowed to mark questions as correct where the response does not match the correct responses pattern. Systems reading statements cannot infer success based on comparison of the Response with the Correct Responses Pattern, nor can they rely on the Correct Responses Pattern always being exhaustive. 
```

Comment:
I totally agree with what you are saying, but we need to be careful about invalidating the usage entirely.  Maybe try starting with something like:

"This specification makes no assertion on how Statements use other properties in combination with the "correct responses pattern" property.  It is possible an activity has results indicating correctness even if this is contrary to what responses and the correct responses pattern provide."